Firmware/mmu2_protocol_logic.cpp

```diff
@@ -124,8 +124,6 @@ struct OldMMUFWDetector {
             return State::MatchingPart;
         } else if(ok == 1 && c == 'k'){
             ++ok;
-            return State::MatchingPart;
-        } else if(ok == 2 && c == '\n'){
```

Comment:
removing this part is a bit speculative, but since we don't use any lowercase letters in the protocol yet, it seems to be safe